Documentation/installation/k8s-install-helm.rst

```diff
@@ -161,11 +161,7 @@ Install Cilium
                      && iptables -t nat -F AWS-CONNMARK-CHAIN-0 \\
                      && iptables -t nat -F AWS-CONNMARK-CHAIN-1
 
-         Some Linux distributions use a different interface naming convention.
-         If you use masquerading with the option ``egressMasqueradeInterfaces=eth+``,
-         remember to replace ``eth+`` with the proper interface name. For
-         reference, Amazon Linux 2 uses ``eth+``, whereas Amazon Linux 2023 uses
-         ``ens+``.
+
 
```

Comment:
Nit: No need for three blank lines in a row

Reply:
thanks. I have changed it locally and wait for other feedbacks to push the commits